src/model/core/co2.jl

```diff
@@ -0,0 +1,107 @@
+@doc raw""" 
+
+co2!(EP::Model, inputs::Dict)
+
+This function creates expression to account for CO2 emissions and captured and sequestrated CO2 from thermal generators. It also has the capability to model the negative CO2 emissions from bioenergy with carbon capture and storage. This module will displace the emissions module.
+
+***** Expressions *****
+
+For thermal generators that use fuels that contain CO2 content (e.g., coal, natural gas, and biomass), the CO2 emissions are a function of fuel consumption, CO2 capture rate, and whether the feedstock is biomass. Biomass (e.g., wastes or agriculture resides) derived energy is typically considered to be carbon-neutral because the carbon in the biomass is originated from the atmosphere. When bioenergy is coupled with carbon capture and storage (CCS), it creates negative emissions.
```

Comment:
> Is it treated as zero CO2 except when paired with CCS
Yes! if paired with CCS, it creates negative emissions

> And is the user supposed to give it a nonzero emissions factor?
Yes, for example, in "Generators_data.csv", for a resource that represents biomass with CCS, the inputs must provide "CO2_Capture_Rate" (a fraction),  "Biomass" (either 0 or 1), and "Fuel" type (e.g., delicated_biomass ). Then in fuels_data.csv, we will need to have the co2 content and cost for the delicated_biomass.

Right now, the existing biomass in generator_data.csv is specified as "None", which is probably because most of them are landfill gas/municipal solid waste plants. In the future, if we want to use dedicated bioenergy or dedicated biomass with CCS, we need to assign a cost and CO2 content (if paired with CCS).